texar/torch/modules/encoders/t5_encoder.py

```diff
@@ -0,0 +1,252 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+from torch import nn
+from texar.torch.modules.encoders.encoder_base import EncoderBase
```

Comment:
It is recommended to add a blank line between third-party imports and imports within the current package. Generally imports are formatted like this:
```python
import time  # built-in imports
import sys
# blank line
import torch  # third-party imports
from torch import nn
# blank line
from texar.torch.core import layers  # imports from the current package, i.e. texar
```